README.md

```diff
@@ -56,12 +58,15 @@ Create a postgres database, called datasource <br />
     CONNECTION LIMIT = -1;
   ```
 
-Run the application entry point:
+Run the application entry point - development mode:
+
+
 
 ```console
-$ python3 main.py
+$ uvicorn app.server.api:app --port 8080 --reload
```

Comment:
Replace it with:

```console
$ gunicorn -w 4 -k uvicorn.workers.UvicornWorker app.server.api:app --bind 0.0.0.0:8080
```

Reply:
This doesn't reload the app, we need it for development don't we?

Reply:
Oh ok. That's true.